htdocs/comm/propal/list.php

```diff
@@ -53,6 +53,9 @@
 require_once DOL_DOCUMENT_ROOT.'/core/lib/company.lib.php';
 require_once DOL_DOCUMENT_ROOT.'/comm/propal/class/propal.class.php';
 require_once DOL_DOCUMENT_ROOT.'/projet/class/project.class.php';
+if (isModEnabled('commande')) {
```

Comment:
`isModEnabled('order')`

Reply:
The phan error message is misleading:
```
Warning: list.php: DeprecatedModuleName: Argument 0 function isModEnabled can't have the value "commande" that does not match the expected pattern '/^(?!(?:actioncomm|adherent|adherent_type|banque|categorie|commande|contrat|entrepot|expedition|facture|ficheinter|product_fournisseur_price|product_price|projet|propale|socpeople)$).*/'
```

From what I read 'commande' is part of what is expected. I'm OK to change it to 'order' however, I would like to be sure that it works...